website/docs/dbt-cli/install/from-source.md

```diff
@@ -1,15 +1,16 @@
 ---
-title: "Install from source"
+title: "Install dbt from source"
 id: "from-source"
+description: "You can install dbt Core from its GitHub code source."
 ---
 
-dbt Core is open source software, as are almost all adapter plugins. As such, the codebases are freely available to download and build from source. To do this, you can clone the repositories from GitHub and install the local version using `pip`.
+dbt Core and almost all of its adaptor plugins are open source software. As such, the codebases are freely available to download and build from source. To do this, you can clone the repositories from GitHub, which makes a local copy, and then install the local version using `pip`.
```

Comment:
We spell this as `adapter` almost everywhere else. We've [discussed this before](https://dbt-labs.slack.com/archives/C025NM6P1SL/p1626300345044300), haha

Reply:
D'oh!